task/update-infra-deployments/0.1/update-infra-deployments.yaml

```diff
@@ -202,46 +204,43 @@ spec:
                 return json_output
 
             def create_reset_branch(self):
-                branch = os.environ.get('ORIGIN_REPO').split('/')[-1]
-                target_gh_repo = os.environ.get('TARGET_GH_REPO')
-                target_branch = self._request("GET", f"/repos/{target_gh_repo}/git/refs/heads/{branch}").json()
-                main_branch_sha = self._request("GET", f"/repos/{target_gh_repo}/git/refs/heads/main").json()['object']['sha']
+                branch = ORIGIN_REPO.split('/')[-1]
```

Comment:
What are the values or ORIGIN_REPO here?

Here the **ORIGIN_REPO.split('/')[-1]** refers to branch, but on line 187, we have the same code where it refers to repo name

```
repo_name = ORIGIN_REPO.split('/')[-1]
```

Reply:
Each RHTAP component has its own branch to hold updates in the target repository, which is generally the infra-deployments, and in build-definitions CI, update-infra-deployments task is also used to update enterprise-contract/ec-policies.

Reply:
I see, thanks for the explanation.